packages/asset-swapper/test/exchange_swap_quote_consumer_test.ts

```diff
@@ -39,7 +39,7 @@ describe('ExchangeSwapQuoteConsumer', () => {
     let coinbaseAddress: string;
     let makerAddress: string;
     let takerAddress: string;
-    let fillScenarios: FillScenarios;
```

Comment:
replacing with OrderFactory while we are here, since we plan to get rid of `fill-scenarios` entirely.